src/flatten-stop-routes.js

```diff
@@ -0,0 +1,32 @@
+const turf = require('@turf/helpers');
+
+function flattenStopRoutes(geojson) {
+  const stopIdPointLookup = {};
+  const routeIdLineStringLookup = {};
+
+  for(const feature of geojson.features){
+    if(feature.geometry.type === 'Point'){
+      const routes = feature.properties.routes;
+      feature.properties['route_ids'] = routes.map((route) => route.route_id).join(',');
+      delete feature.properties.routes;
```

Comment:
Why mutate the input GeoJSON like this? It looks like these changes don't affect the output, correct?

Reply:
good catch. Its cruft leftover from when I had this script generating the tileset as well.